collections/_article/release-candidate-godot-4-0-rc-2.md

```diff
@@ -0,0 +1,107 @@
+- GUI: Cleanup Window callbacks before destroying in to avoid callback calls with invalid object ([GH-73239](https://github.com/godotengine/godot/pull/73239)).
+- Import: Fix SceneImportSettings performance issues ([GH-71691](https://github.com/godotengine/godot/pull/71691)).
+- Import: Set Default compression to VRAM uncompressed for LightmapGI ([GH-73136](https://github.com/godotengine/godot/pull/73136)).
+  * This fixes a perceived regression where lightmaps starting being compressed to BPTC, which is slow. You should remove the `.import` files for your baked lightmaps generated with beta 17 or RC 1 if you want to apply this automatically (or change the compression option in the Import tab). Note that compressing lightmaps is recommended once you're happy with the results, though note that RC 2 still has a known issue exporting BPTC compressed files, which will be fixed in RC 3.
+- Import: Prevent recursive importing (hack) ([GH-73214](https://github.com/godotengine/godot/pull/73214)).
+- Multiplayer: Fix MultiplayerSpawner not connecting to `child_entered_tree` ([GH-73161](https://github.com/godotengine/godot/pull/73161)).
+- Navigation: Fix missing avoidance updates when using same velocity ([GH-72947](https://github.com/godotengine/godot/pull/72947)).
+- Navigation: Fix navigation support for multilayered TileMaps ([GH-73018](https://github.com/godotengine/godot/pull/73018)).
+- Rendering: Add render buffer support to screen space effects ([GH-72838](https://github.com/godotengine/godot/pull/72838)).
+- Rendering: OpenGL: Handle 0 exponent in float/half conversion ([GH-72914](https://github.com/godotengine/godot/pull/72914)).
+- Rendering: OpenGL: Properly reset blend mode when resetting canvas in compatibility renderer ([GH-73006](https://github.com/godotengine/godot/pull/73006)).
+- Rendering: Vulkan: Do clear request before reading from render target when using `CANVAS_BG` ([GH-73055](https://github.com/godotengine/godot/pull/73055)).
+- Rendering: Vulkan: Make draw command labels thread safe ([GH-73057](https://github.com/godotengine/godot/pull/73057)).
+- Rendering: OpenGL: Store blend mode between CanvasItems to preserve batching ([GH-73255](https://github.com/godotengine/godot/pull/73255)).
+- Rendering: Notify mesh surface when `render_priority` changes ([GH-73263](https://github.com/godotengine/godot/pull/73263)).
+- Rendering: Bias octahedral tangent y axis to avoid errors around 0 ([GH-73265](https://github.com/godotengine/godot/pull/73265)).
+- Shaders: Avoid inserting `screen_texture` compatibility code when using `textureSize` ([GH-73060](https://github.com/godotengine/godot/pull/73060)).
+- Thirdparty: Update libwebp to upstream 1.3.0 ([GH-72045](https://github.com/godotengine/godot/pull/72045)).
+
+This release is built from commit [d2699dc7a](https://github.com/godotengine/godot/commit/d2699dc7ab96fbd75faccc1f32f55baebf1d84dc).
+
+<a id="downloads"></a>
```

Comment:
Never caught this one but I think this should be replaced with a `div` or other tag. `<a>` tags should always have an `href` value.
```suggestion
<div id="downloads"></div>
```

Reply:
Fixed!